contracts/core/FreeBets/FreeBetsHolder.sol

```diff
@@ -317,6 +355,96 @@ contract FreeBetsHolder is Initializable, ProxyOwned, ProxyPausable, ProxyReentr
         return resolvedTicketsPerUser[_user].elements.length;
     }
 
+    /// @notice checks if a free bet is valid
+    /// @param _user the address of the user
+    /// @param _collateral the address of the collateral
+    /// @return isValid true if the free bet is valid, false otherwise
+    /// @return timeToExpiration the time to expiration of the free bet, 0 if the free bet is not valid
+    function isFreeBetValid(address _user, address _collateral) external view returns (bool isValid, uint timeToExpiration) {
+        (isValid, timeToExpiration) = _isFreeBetValid(_user, _collateral);
+    }
+
+    /// @notice get users with free bet per collateral
+    /// @param _collateral the address of the collateral
+    /// @param _index the start index
+    /// @param _pageSize the page size
+    /// @return users
+    function getUsersWithFreeBetPerCollateral(
+        address _collateral,
+        uint _index,
+        uint _pageSize
+    ) external view returns (address[] memory) {
+        return usersWithFreeBetPerCollateral[_collateral].getPage(_index, _pageSize);
+    }
+
+    /// @notice get number of users with free bet per collateral
+    /// @param _collateral the address of the collateral
+    /// @return number of users
+    function numOfUsersWithFreeBetPerCollateral(address _collateral) external view returns (uint) {
+        return usersWithFreeBetPerCollateral[_collateral].elements.length;
+    }
+
+    /// @notice get users with valid free bet per collateral
+    /// @param _collateral the address of the collateral
+    /// @param _index the start index
+    /// @param _pageSize the page size
+    /// @return users
+    function getUsersWithValidFreeBetPerCollateral(
```

Comment:
do you really need both methods `getUsersWithValidFreeBetPerCollateral` and `getUsersWithInvalidFreeBetPerCollateral`?
Why not return all users with the information whether his freeBet is Valid or not and handle the logic that separates those with invalid freebets offchain?